tests/test_dome_csc.py

```diff
@@ -37,23 +71,29 @@ def basic_make_csc(self, initial_state, config_dir, simulation_mode):
     async def test_do_stopLouvers(self):
         pass
 
-    async def test_do_closeShutter(self):
+    async def test_do_openShutter(self):
         pass
 
-    async def test_do_openShutter(self):
+    async def test_do_closeShutter(self):
         pass
 
     async def test_do_stopShutter(self):
         pass
 
-    async def test_do_stopAz(self):
+    async def test_do_park(self):
         pass
 
-    async def test_do_stopEl(self):
+    async def test_do_setTemperature(self):
         pass
 
-    async def test_do_stop(self):
+    async def test_config(self):
         pass
 
-    async def test_do_setTemperature(self):
+    async def test_fans(self):
+        pass
+
+    async def test_inflate(self):
+        pass
+
+    async def test_status(self):
```

Comment:
I concerned that all these null tests will make it difficult to figure out which tests are not yet implemented (especially once you have implemented most of them). Consider replacing `pass` with `raise unittest.SkipTest("Not implemented")` or something similar.

Reply:
Done